ground/src/main/java/com/google/android/ground/ui/home/SignOutConfirmationDialog.kt

```diff
@@ -64,5 +76,27 @@ fun SignOutConfirmationDialog(
           }
         },
       )
+    openSignOutDialog.value ->
+      AlertDialog(
+        onDismissRequest = { dismissSignoutDialog() },
+        title = { userDetails?.displayName?.let { Text(it) } },
```

Comment:
You can just call this `user` instead of `userDetails` since it's non ambiguous here.